package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/preset-env": "^7.22.9",
     "@cypress/webpack-preprocessor": "^5.17.1",
```

Comment:
Just noticing now that we have this as a package dependency, Should we should have cypress dependency as a dependency for this package? not all projects use cypress so hence why I ask.
cc @david-mooney